package.json

```diff
@@ -685,7 +697,239 @@
           "when": "swift.hasPackage"
         }
       ]
-    }
+    },
+    "breakpoints": [
+      {
+        "language": "arm"
```

Comment:
Seems like a typo?  I suspect it was copying the in-tree plugin, which lists Ada (which IMO we shouldn't bother with as I'm not sure if its ever been tested).

Reply:
Again just copied from the original package.json